packages/hoppscotch-sh-admin/src/composables/useConfigHandler.ts

```diff
@@ -149,74 +152,45 @@ export function useConfigHandler(updatedConfigs?: ServerConfigs) {
     );
   };
 
-  /**
-   * The updated configs are transformed into a format that can be used by the mutations
-   */
-
-  const workingUpdatedConfigs: ConfigTransform[] = [
-    {
-      config: GOOGLE_CONFIGS,
-      enabled: updatedConfigs?.providers.google.enabled,
-      fields: updatedConfigs?.providers.google.fields,
-    },
-    {
-      config: GITHUB_CONFIGS,
-      enabled: updatedConfigs?.providers.github.enabled,
-      fields: updatedConfigs?.providers.github.fields,
-    },
-    {
-      config: MICROSOFT_CONFIGS,
-      enabled: updatedConfigs?.providers.microsoft.enabled,
-      fields: updatedConfigs?.providers.microsoft.fields,
-    },
-    {
-      config: MAIL_CONFIGS,
-      enabled: updatedConfigs?.mailConfigs.enabled,
-      fields: updatedConfigs?.mailConfigs.fields,
-    },
-  ];
+  // Transforming the working configs back into the format required by the mutations
+  const transformInfraConfigs = () => {
+    const updatedWorkingConfigs: ConfigTransform[] = [
+      {
+        config: GOOGLE_CONFIGS,
+        enabled: updatedConfigs?.providers.google.enabled,
+        fields: updatedConfigs?.providers.google.fields,
+      },
+      {
+        config: GITHUB_CONFIGS,
+        enabled: updatedConfigs?.providers.github.enabled,
+        fields: updatedConfigs?.providers.github.fields,
+      },
+      {
+        config: MICROSOFT_CONFIGS,
+        enabled: updatedConfigs?.providers.microsoft.enabled,
+        fields: updatedConfigs?.providers.microsoft.fields,
+      },
+      {
+        config: MAIL_CONFIGS,
+        enabled: updatedConfigs?.mailConfigs.enabled,
+        fields: updatedConfigs?.mailConfigs.fields,
+      },
+    ];
 
-  // Push or filter the configs based on the enabled condition
-  const transformInfraConfigs = (workingConfigs: ConfigTransform[]) => {
```

Comment:
Supplying this as an arg is unnecessary, hence scoped locally within the function definition.